static/js/containers/DashboardPage.js

```diff
@@ -126,6 +135,26 @@ class DashboardPage extends React.Component {
     }
   };
 
+  updateRequirements = () => {
+    this.fetchDashboard();
+    this.fetchCoursePrices();
+    this.handleOrderStatus();
+  };
+
+  fetchDashboard() {
+    const { dashboard, dispatch } = this.props;
+    if (dashboard.fetchStatus === undefined) {
```

Comment:
As far as I can tell this is exactly the check that `_.isUndefined` makes anyhow: https://github.com/lodash/lodash/blob/master/lodash.js#L12197